src/vivarium_public_health/disease/model.py

```diff
@@ -31,9 +31,13 @@ def __init__(self, cause, initial_state=None, get_data_functions=None, cause_typ
             self._get_data_functions['csmr'] = lambda cause, builder: builder.data.load(
                 f"{self.cause_type}.{cause}.cause_specific_mortality")
 
+    @property
+    def name(self):
+        return f"disease_model.{self.cause}"
+
     @property
     def condition(self):
-        return self.state_column
+        return self.cause
```

Comment:
these are synonyms but I felt it was confusing to rely on the attribute of super, this was a little more clear to me

Reply:
hmm I agree it was confusing. The only thing I'm a little worried about is technically self.cause and self.state_column are synonyms only because we both assign them to equal cause in init. If we ever change that, it could be problematic, because condition is used below as the creates_columns arg to initializes_simulants (line 56). But it does seem unnecessary to have state_column, cause, and condition all now be the same thing.

Reply:
yeah I'd like to close this loop. It seems like condition should just mean cause b/c I don't know the intent behind making this property in the first place throughout this whole PR i've wanted to try and limit reaching into the super class where not totally necessary to make the inheritance less gnarly